rust/azure_iot_operations_services/src/azure_device_registry/models/asset.rs

```diff
@@ -350,13 +350,13 @@ pub struct DiscoveredManagementGroupAction {
 /// Represents a stream for an asset.
 #[derive(Clone, Debug)]
 pub struct Stream {
-    /// Destinations for a stream.
+    /// Destinations for a Stream.
```

Comment:
Capitalized or not? Inconsistent, even within this struct.

Reply:
there were a lot of changes in the 6-5 dtdl update specific to capitalization changes, so I'm just matching it exactly